target/linux/ramips/dts/rt3662_engenius_esr600h.dts

```diff
@@ -161,9 +161,9 @@
 &pci1 {
 	status = "okay";
 
-	wifi@0,1,0 {
+	wifi@0,0 {
```

Comment:
given that the `reg` value is 0x10000, shouldn't it be wifi@1000,0?

Reply:
good question. dtc expects 0,0 here. anything else will error.

Reply:
May I ask for the command(s) you used to compile the dts?

Reply:
https://github.com/neheb/openwrt/commit/b87f8497305bd458ff2c4e31f708c67d59482f58

actually remove all of them.

Reply:
what about 0,0,0 does that error too?

Reply:
yes

Reply:
IIRC, any value except 0x1000 errors with dtc. I think because bus-range for the pcie node is <1 255>. It's <0 255> in other SoCs.

all files that include rt3883.dtsi use the same values for wifi reg as of this commit. Best to be consistent.